src/dumpContent.ts

```diff
@@ -0,0 +1,110 @@
+#!npx vite-node
+import fs from 'fs'
+import { promisify } from 'util'
+
+type ObjData = { _id: string }
+type SearchData = { continuation?: string; objs: ObjData[] }
+type BlobsData = { private_access: { get: { url: string } } }
+
+const DIRECTORY = 'contentDump'
+
+const API_KEY = process.env.SCRIVITO_API_KEY
+const INSTANCE_ID = process.env.SCRIVITO_TENANT
+
+if (INSTANCE_ID && API_KEY) {
+  clearDump()
+  await dumpContent()
+  console.log('\n✅ Dump complete.')
+} else {
+  console.error(
+    'Please provide SCRIVITO_TENANT and SCRIVITO_API_KEY of the instance to dump.',
+  )
+  process.exitCode = -1
+}
+
+function clearDump() {
+  fs.rmSync(DIRECTORY, { force: true, recursive: true })
+  fs.mkdirSync(`${DIRECTORY}/objs`, { recursive: true })
+  fs.mkdirSync(`${DIRECTORY}/binaries`, { recursive: true })
+}
+
+async function dumpContent() {
+  let continuation: string | undefined
+
+  do {
+    const data: SearchData = await fetchJson<SearchData>(
+      `workspaces/published/objs/search`,
+      JSON.stringify({
+        continuation,
+        include_objs: true,
+        options: { site_aware: true },
+        size: 10,
+      }),
+    )
+
+    for (const objData of data.objs) await dumpObjAndBinaries(objData)
+
+    continuation = data.continuation
+  } while (continuation)
+}
+
+async function dumpObjAndBinaries(objData: ObjData) {
+  await dumpBinaries(objData)
+  dumpObj(objData)
+}
+
+async function dumpBinaries(data: unknown) {
+  if (!data || typeof data !== 'object') return
+  if (isBinaryData(data)) await dumpBinary(data[1].id)
+  else for (const d of Object.values(data)) await dumpBinaries(d)
+}
+
+function isBinaryData(data: unknown): data is ['binary', { id: string }] {
+  return (
+    Array.isArray(data) &&
+    data.length === 2 &&
+    data[0] === 'binary' &&
+    typeof data[1] === 'object' &&
+    data[1].id
+  )
```

Comment:
I also suggest to make it more robust:

```suggestion
  if (!Array.isArray(data)) return false

  const [attributeType, attributeValue] = data as unknown[]
  return (
    attributeType === 'binary' &&
    isObject(attributeValue) &&
    typeof (attributeValue as { id: unknown }).id === 'string'
  )
```

Otherwise it might throw a TypeError:

```js
isBinaryData(['binary', null])
// => Uncaught TypeError: Cannot read properties of null (reading 'id')
```

And also returns "truish" for other types as well:

```js
isBinaryData(['binary', { id: { notA: 'string, but an object' } }])
// => {notA: 'string, but an object'}
```